### PR TITLE
created shahid.json

### PR DESCRIPTION
**Pull request by shahidhussain07**

### CODE REVIEW

**Comment by shahidhussain07** [comment]
I understand

File: public/data/shahid.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/shahidhussain07",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/shahidHtwts",
+      "icon": "twitter"
+    }
```
Comment (Panquesito7) [inline public/data/shahid.json:16]: There's a missing comma here.
```suggestion
    },
```

File: public/data/shahid.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://instagram.com/ig_shahidhussain",
+      "icon": "instagram"
+    }
```
Comment (Panquesito7) [inline public/data/shahid.json:21]: A missing comma here as well.
```suggestion
    },
```

**Comment by shahidhussain07** [comment]
Thanks for addressing! <3.

File: public/data/shahid.json (hunk 2 above)
Comment (shahidhussain07) [reply public/data/shahid.json]: Thanks for addressing <3

Comment (shahidhussain07) [reply public/data/shahid.json]: I have updated my JSON file as you've suggested.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good to me! 🚀

File: public/data/shahid.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      "url": "https://twitter.com/shahidHtwts",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://instagram.com/ig_shahidhussain",
+      "icon": "instagram"
+    },
+    {
+      "name": "Connect with me On Linkedin",
```
Comment (Panquesito7) [inline public/data/shahid.json:23]: I recommend capitalizing the `i` here. What do you think?
```suggestion
      "name": "Connect with me On LinkedIn",
```

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks for contributing! 🎉

**Pull request merged**